integration/tests/conftest.py

```diff
@@ -573,3 +573,21 @@ def counter_resource_address(call_solana_caller, accounts, web3_client):
     )
     web3_client.send_transaction(accounts[0], instruction_tx)
     yield call_solana_caller.functions.getResourceAddress(salt).call()
+
+
+@pytest.fixture(scope="class")
+def block_timestamp_contracts(web3_client, accounts):
+    contract_a, _ = web3_client.deploy_and_get_contract("common/Block.sol", "0.8.10", accounts[0], contract_name="BlockTimestamp")
+    contract_b, _ = web3_client.deploy_and_get_contract("common/Block.sol", "0.8.10", accounts[0], contract_name="BlockTimestampCaller")
+    contract_c, receipt = web3_client.deploy_and_get_contract("common/Block.sol", "0.8.10", accounts[0], contract_name="BlockTimestampInitializer")
+    contract_d, _ = web3_client.deploy_and_get_contract("common/Block.sol", "0.8.10", accounts[0], contract_name="BlockTimestampMapping")
+    return contract_a, contract_b, (contract_c, receipt), contract_d
+
+
+@pytest.fixture(scope="class")
+def block_number_contracts(web3_client, accounts):
```

Comment:
please move fixtures to class where it's used if they are needed only in 1 place

Reply:
Fixed ✅